composer.json

```diff
@@ -10,10 +10,10 @@
         }
     ],
     "require": {
-        "php":  "~7.0"
+        "php":  "~7.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7"
+        "phpunit/phpunit": "^6.5"
```

Comment:
Can we move to `^7`?

Reply:
It sounds good. I will set the PHPUnit for defining `^7`.